docker/storm/tests/functional_tests/running_test.yaml

```diff
@@ -0,0 +1,39 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+setup:
+- command: [docker, run, -d, --name, storm-$UNIQUE-id, '$IMAGE']
+- command: [sleep, 30s]
+
+teardown:
+- command: [docker, stop, storm-$UNIQUE-id]
+- command: [docker, rm, storm-$UNIQUE-id]
+
+target: storm-$UNIQUE-id
+tests:
+- name: Test health status
+  command: [curl, -I, -s, -L, 'http://localhost:8080']
+  expect:
+    stdout:
+      matches: '200 OK'
+- name: Test Storm homepage
+  command: [curl, -L, 'http://localhost:8080']
+  expect:
+    stdout:
+      matches: 'Storm UI'
+- name: Can connect to ZooKeeper
+  command: [/apache-zookeeper-3.8.0-bin/bin/zkCli.sh, ls, /zookeeper]
```

Comment:
Try to make it more generic, not version-oriented. using symlink or glob pattern for calling the sh.

Reply:
folder changed